airbyte-integrations/connectors/source-stripe-singer/setup.py

```diff
@@ -32,7 +32,7 @@
     packages=find_packages(),
     package_data={"": ["*.json"]},
     install_requires=[
-        "tap-stripe==1.4.4",
+        "tap-stripe @ https://github.com/yevhenii-ldv/tap-stripe/tarball/master#egg=tap-stripe-1.4.5",
```

Comment:
can you use github.com/airbytehq/tap-stripe ? it should be the same as your fork.

Reply:
updated